src/modules/tasks.js

```diff
@@ -0,0 +1,27 @@
+export default class ToDoTasks {
+  constructor() {
+    this.list = [];
+  }
+
+  addTask(taskDescription) {
+    const task = {
+      description: taskDescription,
+      completed: false,
+      index: this.list.length + 1,
+    };
+    this.list.push(task);
+  }
+
+  removeTask(index) {
+    this.list = this.list.filter((item) => item.index !== index);
+    // update indices
+    for (let i = 0; i < this.list.length; i += 1) {
+      this.list[i].index = i + 1;
+    }
+  }
```

Comment:
Kindly, implement `Clear all completed` button as shown [here](https://www.youtube.com/watch?v=AcUd-_Yjjqg) in the video the user can remove all the completed items. Also, implement the CSS when the user checks the task completed as the video.

Reply:
Those changes are not the requirements of this part of the project. Those are the requirements for tomorrow's part of the project.

Reply:
@diegoyon, I see now it's tomorrow's project. You can submit what you have and the next reviewer will approve it.